sensor_msgs/PointCloud2 cloth_cloud
---
normal_descriptor_node/heat_map wrinkle_map
